backend/app/models/campaignAction.js

```diff
@@ -1,13 +1,32 @@
 const mongoose = require('mongoose')
 const moment = require('moment')
 const Schema = mongoose.Schema
+const ObjectId = mongoose.Types.ObjectId
 
 const campaignActionSchema = new Schema({
   title: String,
   createdAt: { type: Date, default: () => moment.utc().toDate() },
   sent: { type: Boolean, default: false },
   sentAt: Date,
-  campaign: { type: Schema.Types.ObjectId, ref: 'Campaign' }
+  campaign: { type: Schema.Types.ObjectId, ref: 'Campaign' },
+
+  // TARGETING
+  targetingType: { type: String, enum: ['segmenting', 'borrowed'] },
```

Comment:
going to merge this in now because I don't have time to test at the moment
but would be happy to do a find+replace all to replace borrowed with inherited in a separate PR
i'll add an issue